proj/src/components/statistics/statistics.js

```diff
@@ -1,16 +1,19 @@
 import React, { Component } from 'react'
+import { connect } from 'react-redux';
 import styled from 'styled-components'
 import { flow, map, keys, flatten, uniq, filter } from 'lodash/fp';
 import {BarChart, Bar, XAxis, YAxis, CartesianGrid, Tooltip, Legend} from 'recharts';
 
+import { fetchData } from 'actions/statisticsActions'
+
 const Wrapper = styled.div`
   height: 100vh;
   display: flex;
   justify-content: center;
   align-items: center;
 `
 
-export default class Statistics extends Component {
+class Statistics extends Component {
 
   state = {
```

Comment:
let's start cleaning up component state. we have redux and mimic now so we need to remove all the junk